### DISCUSSION TITLE
文档为什么没有一些常用的插件列表，普通应用怎么添加微前端也没说明白，只说了max内置

### DISCUSSION THREAD

**Discussion by 2531800823** [umijs/umi#10045] [Ideas]
希望优化官方文档，写清楚一点！！

**Comment by fz6m** [comment]
你说的对。

**Reply by fz6m** [reply]
在 #10049 解决。

**Comment by ghost** [comment]
https://umijs.org/docs/max/mf#module-federation-%E6%8F%92%E4%BB%B6 @2531800823  微前端这个里面有

**Reply by 2531800823** [reply]
这个是用的模块联邦，不是qiankun的微前端
